data-raw/R2R_TMH_habitat_inputs/tmh_helper_functions.R

```diff
@@ -14,20 +14,199 @@ gis_calcs <- function(data) {
     filter(!is.na(river_length_feet)) 
 }
 
-existing_flow_cfs <- function(habitat_type, watershed_input, bypass = FALSE) {
+format_all_hab_data_long <- function(tmh_data_formatting) {
+  tmh_data_formatting |> 
+    rename(spwn_acres_max = max_spawning_acres,
+           rear_acres_max = max_rearing_acres, 
+           flood_acres_max = max_floodplain_acres) |> 
+    pivot_longer(cols = c(spwn_acres_max:flood_acres_max), names_to = "metric") |>
+    separate(metric, c('hab', 'unit', 'lifestage'), "_") 
+}
+
+all_existing_and_tmh_data_fun <- function(species) {
```

Comment:
implemented

Reply:
This wasn't exactly what I had in mind. The goal with one file is that you only have to read it in once so you could read in and filter to species instead of keeping the species separate. Please update later